### PR TITLE
Feature/fc 2569 add docs

### PR DESCRIPTION
**Pull request by mrchrisjoy**
**QA** https://foundcareers.atlassian.net/browse/FC-2569

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,247 +65,357 @@ Here's an example store that `@foundcareers/redux-entity` can work with, with th
...
 import {createCollectionState} from '@foundcareers/redux-entity';
-
 const stateWithStandardPagination = createCollectionState({
-  previouslySelectedEntityId: null,
+ previouslySelectedEntityId: null,
 }, false);
 
 const options = {
-  useCursor: true
+ useCursor: tru
```
Comment (mcelliott) [inline README.md]: `true`

File: README.md [hunk 2]
```diff
@@ -61,247 +65,357 @@ Here's an example store that `@foundcareers/redux-entity` can work with, with th
...
+
 const stateWithMetaPagination = createCollectionState({}, options);
-```
-Resulting in the following initial collection objects:
-```
-// stateWithStandardPagination
+
+// Returns the following collection state objects
+
+// console.log(stateWithStandardPagination)
```
Comment (mcelliott) [inline README.md]: remove these console.logs

File: src/factory.js [hunk 3]
```diff
@@ -15,6 +17,50 @@ const createDefaultMeta = () => ({
...
+ * @param {Object} options Configuration object.
+ * @param {boolean} options.useCursor Set to `true` to use cursor meta.
+ * @example
+ * import {createCollectionState} from '@foundcareers/redux-entity';
+ * const stateWithStandardPagination = createCollectionState({
+ *  previouslySelectedEntityId: null,
+ * }, false);
+ * 
+ * const options = {
+ *  useCursor: tru
```
Comment (mcelliott) [inline src/factory.js]: `true`

File: README.md [hunk 4]
```diff
@@ -61,247 +65,357 @@ Here's an example store that `@foundcareers/redux-entity` can work with, with th
...
-    startCursor: null,
-  },
-  selectedEntityId: null,  
+ entities: {},
+ meta: {
+   endCursor: null,
+   hasNextPage: null,
+   startCursor: null
+ },
+ selectedEntityId: null,
```
Comment (mcelliott) [inline README.md]:
```
import {createCollectionState} from '@foundcareers/redux-entity';
```
State with Standard Pagination:
```
const stateWithStandardPagination = createCollectionState({
  previouslySelectedEntityId: null,
}, false);
```
Output:
```
{
  entities: {},
  meta: {
    currentPage: 0,
    nextPage: 0,
    prevPage: null,
    totalPages: 0,
    totalCount: 0
  },
  selectedEntityId: null
}
```
State with Cursor Pagination:
```
const stateWithCursorPagination = createCollectionState({}, {useCursor: true});
```
Output:
```
{
  entities: {},
  meta: {
    endCursor: null,
    hasNextPage: null,
    startCursor: null
  },
  selectedEntityId: null,  
}
```

Comment (mrchrisjoy) [reply README.md]: Updated JSDocs

File: src/factory.js [hunk 5]
```diff
@@ -15,6 +17,47 @@ const createDefaultMeta = () => ({
...
+ * @memberof Factories
+ * @param {Object} state Object that's spread into the collection state.
+ * @param {Object} options Configuration object.
+ * @param {boolean} options.useCursor Set to `true` to use cursor meta.
+ * @example
+ * import {createCollectionState} from '@foundcareers/redux-entity';
+ * 
+ * // State with Standard Pagination:
+ * const stateWithStandardPagination = createCollectionState({
+ *  previouslySelectedEntityId: null,
```
Comment (mcelliott) [inline src/factory.js]: I think this will appear in the output as well.

Comment (mrchrisjoy) [reply src/factory.js]: Ah yes. Thanks, I've just added that.

**Pull request merged**